core/src/main/resources/config.yml

```diff
@@ -5,6 +5,11 @@
 #
 # GitHub: https://github.com/GeyserMC/Geyser
 # Discord: https://discord.geysermc.org/
+# Wiki: https://wiki.geysermc.org/
+#
+# DISCLAIMER: See https://wiki.geysermc.org/geyser/setup/ for the setup guide. Many video tutorials are outdated.
```

Comment:
DISCLAIMER isn't the right tone, I think. Maybe NOTICE instead?

Reply:
fits better, yep
caps are okay, right?